misc/openlane/1439.patch

```diff
@@ -0,0 +1,29 @@
+From e57dc500ef22047e2d57c1c4bcae83260f6d3672 Mon Sep 17 00:00:00 2001
```

Comment:
```suggestion
# From e57dc500ef22047e2d57c1c4bcae83260f6d3672 Mon Sep 17 00:00:00 2001
```
Currently Conda-build fails to apply the patch because of the header. Please add a hash or even just a space at the beginning of this line to solve the problem.